platform/ios/Makefile

```diff
@@ -182,7 +189,7 @@ ios-uitest: $(IOS_PROJ_PATH)
 	echo "You can review uitest results:  open $(IOS_OUTPUT_PATH)/Logs/Test"
 
 .PHONY: ios-test
-ios-test: $(IOS_PROJ_PATH)
+ios-test: $(IOS_PROJ_PATH) style-code
```

Comment:
This also needs to be added to `document`. Otherwise, the generated jazzy docset will omit all the runtime styling classes that developers should have access to.